### DISCUSSION TITLE
How to report a repo (not an app) for having a terribly offensive name?

### DISCUSSION THREAD

**Discussion by paulowens** [community/community#31949] [New to GitHub]
I stumbled across a repo which has an appallingly offensive name (the repo name trivialises violence against a minority group). I realise that these things are subjective and it could very well be me that's out of step but I feel it would be remiss of me to at least report the repo to support to let them decide. I have tried using the [report abuse page](https://support.github.com/contact/report-abuse?category=report-abuse&report=other&report_type=unspecified) but nothing happens when I click the submit button (after having solved the captcha). I am trying to use the "_I don't see my problem here_" option as it seems the most appropriate. Any advice on how to proceed?

**Comment by atomisadev** [comment]
If you feel that the "Report Abuse" button isn't working, contact GitHub Support. :)

Let them know about the offensive repository so they can take appropriate action.
